src/comments.jsx

```diff
@@ -0,0 +1,39 @@
+import React, {useEffect, useState} from 'react';
+import { getCommentsById } from './utils/getCommentsById';
+
+const Comments = ({id}) => {
+    const [comments, setComments] = useState(null);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        getCommentsById(id).then((data) => {
+            setComments(data)
+            setIsLoading(false);
+        }).catch((err) => {
+            console.log(err)
+            setIsLoading(false)
+        })
+    }, [id])
+    if (isLoading) {
+        return (
```

Comment:
We handle loading but we don't handle 0 comments on an article. How could we handle this case as well?